.gitignore

```diff
@@ -57,3 +57,5 @@ typings/
 # dotenv environment variables file
 .env
 
+#package-lock.json
+package-lock.json
```

Comment:
https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5
TL;DR: Nên commit package-lock.json, mục đích chính là để các thành viên trong nhóm dev sử dụng cùng phiên bản của các dependencies